framework/python/src/common/risk_profile.py

```diff
@@ -95,6 +96,17 @@ def get_file_path(self):
     return os.path.join(PROFILES_PATH,
                         self.name + '.json')
 
+  def check_status(self):
+    """Checks if the profile has expired
+    and updates the status to Expired if required."""
+
+    created_date = self.created.timestamp()
+
+    today = datetime.now().timestamp()
+
+    if created_date < (today - SECONDS_IN_YEAR):
+      self.status = 'Expired'
+
```

Comment:
Use _expired method directly to avoid using seconds in year

```
def check_status(self):
    """Checks if the profile has expired
    and updates the status to Expired if required."""
    if self._expired():
      self.status = 'Expired'
```

Reply:
Still using seconds in year instead of directly calling the expired method, which already handles this check.  Seconds in year is inaccurate as it does not account for leap years, should not be used.

If we are going the route of dynamically resolving the status, we should likely privatize the status property and switch all references to a get_status() method call which makes this determination every time.